homeassistant/components/tradfri/base_class.py

```diff
@@ -39,8 +39,7 @@ def __init__(self, device, api, gateway_id):
         self._device_control = None
         self._device_data = None
         self._gateway_id = gateway_id
-        self._name = None
-        self._unique_id = None
+        self._attr_should_poll = False
```

Comment:
This is static, move them to class attributes on the `TradfriBaseClass` class

Reply:
Done, funny thing is that about a week ago, I was asked to remove a class variable 😬